.github/workflows/release.yaml

```diff
@@ -0,0 +1,73 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.15.3'
+      - run: make test
+      - uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
+          args: --timeout 5m
+        env:
+          GOROOT: ""
+      - uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }} #required
+          file: ./coverage.out
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout repo
+        uses: actions/checkout@master
+      - 
+        name: Build image tag
+        run: echo "IMAGE_TAG=v$(cat ./VERSION)" >> $GITHUB_ENV 
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GH_USERNAME }}
+          password: ${{ secrets.GH_PAT }}
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          platforms: linux/amd64,linux/arm64,linux/arm
+          push: true
+          tags: |
+            argoproj-labs/argocd-notifications:${{ env.IMAGE_TAG }}
+            argoproj-labs/argocd-notifications:latest
```

Comment:
Ops, I just realized that I incorrectly asked to avoid pushing the latest tag as part of CI workflow https://github.com/argoproj-labs/argocd-notifications/pull/190 . Sorry

Typically `latest` tag has latest code. @mylesagray , do you mind to move `ghcr.io/argoproj-labs/argocd-notifications:latest` and `argoproj-labs/argocd-notifications:latest` tags to https://github.com/argoproj-labs/argocd-notifications/blob/master/.github/workflows/ci.yaml#L69 ?

Reply:
@alexmt I've made those changes to the PR - So I can understand the desired outcome:

- CI builds on every push to master and _always_ tags latest (no discrete commit based hash tags desired here?)
- Release builds are manual only - and will derive their tag appending `v` to the output of `$(cat ./VERSION)`

Is that what you're after?

Thoughts on adding a tag for commit hash to the CI images too in addition to latest tag so commits can be easier tested?

Reply:
I think it is a good idea but only for `ghcr.io/argoproj-labs/argocd-notifications` . We are trying to have only officially released images in dockerhub.com/argoproj-labs and dockerhub.com/argoproj .